unstructured/ingest/evaluate.py

```diff
@@ -249,6 +250,55 @@ def measure_table_structure_accuracy_command(
     )
 
 
+@main.command()
+@click.option("--output_dir", type=str, help="Directory to structured output.")
+@click.option("--source_dir", type=str, help="Directory to structured source.")
```

Comment:
Could use clicks [Path type](https://click.palletsprojects.com/en/8.1.x/api/#click.Path) here for more control over the parameter's value e.g. only directory, existance

Reply:
True but this should be applied to all ingest commands then - worth a separate PR.